### DISCUSSION TITLE
webdav cann't adds the  access paths

### DISCUSSION THREAD

**Discussion by yuwan1994** [iterate-ch/cyberduck#15353] [Q&A]

**Comment by ylangisc** [comment]
Don't understand your issue. Can you please rephrase it?

**Comment by levigutt** [comment]
I'm having an issue making a connection to a webdav server that requires a specific path. 
Maybe this is the same issue. 

My issue is that I cannot specify the path when connecting: 
<img width="538" height="332" alt="image" src="https://github.com/user-attachments/assets/fb13f41e-5ea4-4916-b787-c03db9311fc6" />

This used to be possible in an older version: 
<img width="413" height="369" alt="image" src="https://github.com/user-attachments/assets/0d0098d1-2c1f-4139-964e-a89df86183af" />

If I try to add the path in the `Server`-field, I get an error, since it will try to check the DNS for the entire URL (including path) which fails: 
<img width="517" height="278" alt="image" src="https://github.com/user-attachments/assets/a33722f2-0bd4-423f-9ebd-8f59419968dd" />


Am I doing something wrong? Is there a way to get the "more options" section back?

**Reply by AliveDevil** [reply]
There's a discrepancy between Windows and macOS - the Windows "Quick Connect" dialog is expandable, though the macOS one isn't.

However, on macOS you can create a new bookmark (not using the Open Connection-button, but in the Bookmarks-tab view, in the lower left corner is an Add-button, with which you can add more detailed information.

I think we had a ticket for this, but I can't find it currently.
